entrypoint.sh

```diff
@@ -39,12 +40,41 @@ case "$connection_format" in
     ;;
 esac
 
+patch_image() {
+    if copa patch -i "$image" -r ./data/"$report" -t "$patched_tag" "$connection" --timeout "$timeout" "$output"
```

Comment:
```suggestion
if copa patch -i "$image" -r ./data/"$report" -t "$patched_tag" $connection --timeout $timeout $output
```

Reply:
can we try this change? this seems to be one difference between the original command and the current. Otherwise, the changes make sense to me.

Reply:
@ashnamehrotra Thank you for the review 😃.
Yes, we can try this change. Let's see if the build succeeds.